hugo/content/contribute.md

```diff
@@ -3,83 +3,261 @@ title: Contribute
+
+Any contributions are welcome, improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+
+The new places for documentation will probably be here:
+
+- [Mumble GitHub repo - Docs](https://github.com/mumble-voip/mumble/tree/master/docs)
+
+- [Website's GitHub repo - Documentation](https://github.com/mumble-voip/mumble-www/tree/master/hugo/content/documentation)
+
+#### Documentation Issues & Pull requests
+
+Also take a look at open Issues and Pull Requests:
+
+- [Documentation issues in Mumble's GitHub repo](https://github.com/mumble-voip/mumble/labels/documentation)
+
+- [Pull Request with TYPE label "Docs"](https://github.com/mumble-voip/mumble/pulls?q=is%3Apr+is%3Aopen+docs)
+
+- [Documentation issues in Mumble's Website repo](https://github.com/mumble-voip/mumble-www/labels/docs)
+
+---
+
+<!-- 
+**Ice methods**: 
+<!-- outdated? -->
+<!-- Is this here now: https://www.mumble.info/documentation/slice/1.3.0/html/Murmur.html  ? -->
+<!--
+We have a [generated documentation](http://mumble.sourceforge.net/slice/) for exposed Ice methods. Although probably only developers using the doc will be able to understand it, please give us a note on any inconsistencies, errors or needed clarifications if you find them.
+-->
+
+### External Documentation
+
+- **Wikipedia**: Someone that is not a development team member needs to maintain the [article(s) at Wikipedia](https://en.wikipedia.org/wiki/Mumble_%28software%29).
+
+
+## Website
+
+<!-- 
+Todo:
+
+-Link to Writing guidelines (markdown etc.), Planned Content etc. (should be in the Readme.md of the mumble-www repo)
+
+-->
+
+This website has a GitHub repository: [mumble-www](https://github.com/mumble-voip/mumble-www)
+
+You can:
+
+- report [issues](https://github.com/mumble-voip/mumble-www/issues)
+- create [requests](https://github.com/mumble-voip/mumble-www/issues), 
+- create pull requests for changes to the website
+
+
+
+## Community
+
+Be a part of our community and help others!
+
+* Our [issue tracker](https://github.com/mumble-voip/mumble/issues)
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
+* Our [Forums](https://forums.mumble.info/)
+
+# Hosting
+
+Host your own Mumble Server and register it on our Public Server List.
+<!-- add new Server guide (once it's ready) -->
+See our {{< wiki "Running_Murmur" "Server Guide" />}} and [Server Registration details](https://wiki.mumble.info/wiki/Murmur.ini#Server_Registration).
+
+You find a list of hosters on our wiki: {{< wiki Hosters />}}.
+<!-- Is that recent and trustworthy information? Maybe it would be better to have a list that is maintained by the team, so there can be a (little) check before an entry is added to the list -->
```

Comment:
?

Reply:
What kind of check would we be able to perform? We can't rent a server from every hoster just to see if and how well it works :thinking:

Reply:
Two things:
1. It can be a legal problem for you, if you wildly recommend server hosters without checking them at all.
2. The performance or usability is not the main problem; the main problem is whether these services are trustworthy or not. So at least a short look at the website, check for https, certificate details, check for obvious things like wrong spelling everything, should be possible.
Some countries also offer company registration sites, where you can check whether a company exists or not.

All in all, I would recommend to delete this or just recommend bigger, trustworthy services.
If people want to find more hosters, they can search for them themselves.

Reply:
> Users might trust the information listed there, even though there is no security at all.

This is nothing that we can prevent from happening. On the contrary: if we start doing little tests here and there then some users might overestimate the value of these tests and then put more trust in these recommendations than if they had known that this is just a more or less uncurated list.

> It can be a legal problem for you, if you wildly recommend server hosters without checking them at all.

We don't recommend, we list. And we are not responsible for what these hosters do. We only have to act if we know that they are scamming people or something like that.

> the main problem is whether these services are trustworthy or not

Trustworthy can be a bit subjective as well. And even only checking for these kind of things is work that someone has to invest the time to do. Personally I don't have the time to do that as well :shrug: 

> All in all, I would recommend to delete this or just list bigger, trustworthy services.

That sounds reasonable to me. @davidebeatrici @Kissaki @ZeroAbility what's your opinion on this?

Reply:
I deleted the link for now.